ansible/playbooks/roles/upgrade/tasks/kubernetes/patch-kubeadm-apiserver.yml

```
@@ -72,6 +73,12 @@
           {% if version is version('1.21', '>=') and version is version('1.22', '<') %}
               enable-admission-plugins: {{ _kubeadm_supported_admission_plugins }}
           {% endif %}
+            extraVolumes:
+              - name: apiserver-log
+                hostPath: /var/log/kubernetes/audit/
+                mountPath: /var/log/apiserver
```

```suggestion
              - name: kube-apiserver-audit-log
                hostPath: /var/log/kubernetes/audit/
                mountPath: /var/log/apiserver/
```
This path was configurable so far, do you know what would happen if audit log was disabled?
We could look for this value in manifest.yml (from the last apply).

---

It will enable this with rotation, so it's quite safe. I can't be sure, but I think nobody cared about changing default values.